src/pages/signIn/config.ts

```diff
@@ -0,0 +1,55 @@
+export const config = [
+  {
+    id: 'firstName',
+    type: 'text',
+    label: 'First name *',
+    isError: true,
+    onAction: true,
+    isRequired: true,
+    placeholder: 'your name',
+    errorText: 'this field must be filled',
+    optionalType: 'text',
+  },
+  {
+    id: 'secondName',
+    type: 'text',
+    label: 'Second name',
+    onAction: true,
+    isRequired: false,
+    placeholder: 'your second name',
+    optionalType: 'text',
+  },
+  {
+    id: 'email',
+    type: 'email',
+    label: 'Email *',
+    isError: true,
+    isRequired: true,
+    placeholder: 'email',
+    errorText: 'this field must be filled',
+    optionalType: 'email',
+  },
+  {
+    id: 'password',
+    type: 'password',
+    label: 'Password *',
+    isError: true,
+    isRequired: true,
+    placeholder: 'password',
+    errorText: 'this field must be filled',
+    onIconAction: true,
+    leftIcon: 'fas fa-eye',
+    optionalType: 'text',
+  },
+  {
+    id: 'repeatedPassword',
+    type: 'password',
+    label: 'Repeat Password *',
+    isError: true,
+    isRequired: true,
+    placeholder: 'repeat password',
+    errorText: 'this field must be filled',
+    onIconAction: true,
+    optionalType: 'text',
```

Comment:
one type should be, why here i see  optionalType: text ? and in interface you should add `;` on the end